code/modules/ninja/ninja_event.dm

```diff
@@ -78,8 +78,7 @@ Contents:
 
 /proc/create_space_ninja(spawn_loc)
 	var/mob/living/carbon/human/new_ninja = new(spawn_loc)
-	var/datum/character_save/CS = new()//Randomize appearance for the ninja.
-	CS.real_name = "[pick(GLOB.ninja_titles)] [pick(GLOB.ninja_names)]"
-	CS.copy_to(new_ninja)
```

Comment:
this effectively removes appearance randomization and makes ninjas default spawned humans. might be worth adding a randomization?

Reply:
Spawned humans are random